config/auth_conf.xml.sample

```diff
@@ -93,7 +93,15 @@
 <!--        <search-user>cn=jsmith,ou=People,dc=domain,dc=com</search-user>
             <search-password>mysecret</search-password>
 -->
-
+            <!-- Generate username or email from the LDAP results if they are not available from the filter -->
+<!--        <ldap-autogenerate>False</ldap-autogenerate>
```

Comment:
This doesn't seem necessary to me if you add the proper fields to `<search-fields/>` above and use them in `<auto-register-username/>` and `<auto-register-email/>` below.

Reply:
Indeed, this seems to be the result of some confusion on our end. This entry and the corresponding code have been removed.